docker/0.90-2/resources/mms/config.properties

```diff
@@ -0,0 +1,10 @@
+vmargs=-Xmx128m -XX:-UseLargePages -XX:+UseG1GC -XX:MaxMetaspaceSize=32M -XX:MaxDirectMemorySize=10m -XX:+ExitOnOutOfMemoryError
+model_store=$$SAGEMAKER_MMS_MODEL_STORE$$
+load_models=$$SAGEMAKER_MMS_LOAD_MODELS$$
+plugins_path=/tmp/plugins
+inference_address=http://0.0.0.0:$$SAGEMAKER_BIND_TO_PORT$$
+management_address=http://0.0.0.0:$$SAGEMAKER_BIND_TO_PORT$$
+default_workers_per_model=$$SAGEMAKER_NUM_MODEL_WORKERS$$
```

Comment:
With MME mode, the default_workers_per_model should be 1. Or are you forcing SAGEMAKER_NUM_MODEL_WORKERS to be 1?